src/serac/numerics/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ set(numerics_sources
     petsc_solvers.cpp
     )
 
-set(numerics_depends serac_infrastructure serac_functional)
+set(numerics_depends serac_physics)
```

Comment:
Since `serac::Mesh` depends on the state manager, it means it depends on `serac_physics`. So, any test that has a mesh will depend on `serac_physics`.

It might make sense to move these tests from `numerics/tests` to `physics/tests`. That way the dependency tree matches the file structure. But, I have a feeling most tests aren't following this rule anyway and might just be better for a future PR.

Reply:
nvm https://github.com/LLNL/serac/issues/1374